### PR TITLE
https://issues.redhat.com/browse/ACM-14394

### PR DESCRIPTION
**Pull request by dockerymick**

### CODE REVIEW

File: governance/policy_generator.adoc [hunk 1]
```diff
@@ -231,12 +239,20 @@ Note that all the fields in the `policyDefaults` section except for `namespace`
 
 | `policyDefaults.evaluationInterval`
 | Optional
-| Use the parameters `compliant` and `noncompliant` to specify the frequency for a policy to be evaluated when in a particular compliance state. When managed clusters have low CPU resources, the evaluation interval can be increased to reduce CPU usage on the Kubernetes API. These are in the format of durations. For example, `"1h25m3s"` represents 1 hour, 25 minutes, and 3 seconds. These can also be set to "never" to avoid evaluating the policy after it has become a particular compliance state.
+| Specifies the frequency for a policy to be evaluated when it is in a particular compliance state. Use the parameters `compliant` and `noncompliant`. The default value is `"watch"` to leverage Kubernetes API watches instead of polling the Kubernetes API server.
```
Comment (dockerymick) [inline governance/policy_generator.adoc]: @mprahl Are the quotation marks necessary?

File: governance/policy_generator.adoc [hunk 2]
```diff
@@ -231,12 +239,20 @@ Note that all the fields in the `policyDefaults` section except for `namespace`
...
+
+When managed clusters have low resources, the evaluation interval can be set to long polling intervals to reduce CPU and memory usage on the Kubernetes API and policy controller. These are in the format of durations. For example, `"1h25m3s"` represents 1 hour, 25 minutes, and 3 seconds. These can also be set to `"never"` to avoid evaluating the policy after it has become a particular compliance state.
```
Comment (dockerymick) [inline governance/policy_generator.adoc]: Are the quotation marks also necessary here?

File: governance/policy_generator.adoc [hunk 3]
```diff
@@ -231,12 +239,28 @@ Note that all the fields in the `policyDefaults` section except for `namespace`
 
 | `policyDefaults.evaluationInterval`
 | Optional
-| Use the parameters `compliant` and `noncompliant` to specify the frequency for a policy to be evaluated when in a particular compliance state. When managed clusters have low CPU resources, the evaluation interval can be increased to reduce CPU usage on the Kubernetes API. These are in the format of durations. For example, `"1h25m3s"` represents 1 hour, 25 minutes, and 3 seconds. These can also be set to "never" to avoid evaluating the policy after it has become a particular compliance state.
+| Specifies the frequency for a policy to be evaluated when it is in a particular compliance state. Use the parameters `compliant` and `noncompliant`.
+
+When managed clusters have low resources, the evaluation interval can be set to long polling intervals to reduce CPU and memory usage on the Kubernetes API and policy controller. These are in the format of durations. For example, `"1h25m3s"` represents 1 hour, 25 minutes, and 3 seconds. These can also be set to `"never"` to avoid evaluating the policy after it is in a particular compliance state.
```
Comment (mprahl) [inline governance/policy_generator.adoc]: The quotation marks are not required but you can keep them if needed.

Comment (mprahl) [inline governance/policy_generator.adoc]: Could you mention the `watch` option here as well and not just in the `.compliant` and `.noncompliant` subsections?

File: governance/policy_generator.adoc [hunk 4]
```diff
@@ -395,7 +423,19 @@ Note that all the fields in the `policyDefaults` section except for `namespace`
...
+a| Path to a single file, a flat directory of files, or a Kustomize directory relative to the `kustomization.yaml` file. If the directory is a Kustomize directory, the generator runs Kustomize against the directory before generating the policies. If there is a requirement to process Helm charts for the Kustomize directory, set `POLICY_GEN_ENABLE_HELM` to `"true"` in the environment where the policy generator is running to enable Helm for the Policy Generator.
+
+The following manifests are supported:
+
+- Non-root policy type manifests such as `IamPolicy`, `CertificatePolicy`, and `ConfigurationPolicy` that have a `Policy` suffix. The previous manifests are not modified except for patches and are directly added as a `policy-templates` entry of the `Policy` resource .
```
Comment (mprahl) [inline governance/policy_generator.adoc:430]: @dhaiducek do you think we should mention Gatekeeper constraints here as well?

Comment (dhaiducek) [reply governance/policy_generator.adoc:430]: Yes, good call! Looks like we need to update the reference YAML in the repo also. 😄

File: governance/policy_generator.adoc (hunk 3 above)
Comment (dockerymick) [reply governance/policy_generator.adoc]: Not required on the doc side, just wanted to confirm that it wasn't required for the parameter value. Thanks!

Comment (dockerymick) [reply governance/policy_generator.adoc]: Ok no problem. I think it's best to just mention it in the evaluationInterval parameter

File: governance/policy_generator.adoc [hunk 5]
```diff
@@ -231,12 +239,28 @@ Note that all the fields in the `policyDefaults` section except for `namespace`
 
 | `policyDefaults.evaluationInterval`
 | Optional
-| Use the parameters `compliant` and `noncompliant` to specify the frequency for a policy to be evaluated when in a particular compliance state. When managed clusters have low CPU resources, the evaluation interval can be increased to reduce CPU usage on the Kubernetes API. These are in the format of durations. For example, `"1h25m3s"` represents 1 hour, 25 minutes, and 3 seconds. These can also be set to "never" to avoid evaluating the policy after it has become a particular compliance state.
+| Specifies the frequency for a policy to be evaluated when it is in a particular compliance state. Use the parameters `compliant` and `noncompliant`. The default value for the `compliant` and `noncompliant` parameters is `watch` to leverage Kubernetes API watches instead of polling the Kubernetes API server.
```
Comment (dockerymick) [inline governance/policy_generator.adoc:242]: Related to https://issues.redhat.com/browse/ACM-14388

**Comment by dockerymick** [comment]
need to create a new PR because of the merge conflicts. there were file name changes.

**Comment by dockerymick** [comment]
New PR here: https://github.com/stolostron/rhacm-docs/pull/7068/files

**Pull request closed without merging**